src/app/containers/Ad/Canonical/index.jsx

```diff
@@ -70,13 +92,20 @@ const CanonicalAd = ({ slotType, className }) => {
   return (
     <>
       <Helmet>
+        {/* PreLoad Ad Scripts */}
+        {getPreloadUrls(queryString).map(script => (
```

Comment:
@chris-hinds Should we only preload if ads enabled for service? Or is this logic already handled by the fact that we won't even show the Ads container if ads are not enabled?

Reply:
This logic should be handled by the container already